DOCKER_README.md

```diff
@@ -0,0 +1,53 @@
+## Docker Compose User Guide
```

Comment:
Should we maybe move this to a /docs/ folder or something? We have been doing most everything on the Wiki, but I would prefer to do things in the repo.